index.html

```diff
@@ -205,6 +227,19 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
             </p>
           </li>
         </ol>
+
+        <p> To <dfn>determine if crop target is valid</dfn>, given {{cropTarget}} |target| as input, run the following
```

Comment:
Shall we change from determining if a crop target is **valid** to determining if it's **valid input**? That is because any CropTarget minted by the UA is "valid" but it might not be possible to use it in certain contexts or at certain times - e.g. if posted to a frame in another tab.

Reply:
renamed